### ISSUE TITLE
Last bracket for root item doesn't line up

### ISSUE THREAD

**Issue by TheKeeperOfPie** [snappdevelopment/JsonTree#62]
You can see it in the screenshots in the current README, but the view looks like:

```
  ▾ {
    ▾ "root": {
         "inner": 1
      }
}
```

To me it feels like it should line up like

```
  ▾ {
    ▾ "root": {
         "inner": 1
      }
    }
```

But this may be just my personal preference.

I think if you want to fix this, the relevant lines are [this](https://github.com/snappdevelopment/JsonTree/blob/e0f6d75735d9ca80000f556936471373fd203c84/jsontree/src/main/java/com/sebastianneubauer/jsontree/JsonTree.kt#L204) and similar, to use the icon size rather than `0.dp`.

**Comment by snappdevelopment** [comment]
Hi @TheKeeperOfPie 
thanks for reporting this. I think I changed this, because it felt awkward that the last bracket was not left aligned and rather dangling in the air. But maybe it should be like that for consistency to the inner objects.

**Comment by snappdevelopment** [comment]
@TheKeeperOfPie This got fixed in version 2.2.0. Thanks for reporting!
